app/models/manageiq/providers/ibm_terraform/configuration_manager/configured_system.rb

```diff
@@ -1,8 +1,11 @@
 class ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystem < ::ConfiguredSystem
   supports :console
-
   include ProviderObjectMixin
 
+  has_one :hardware, :through => :computer_system
+
+  belongs_to :ext_management_system, :foreign_key => "manager_id"
```

Comment:
These already `belongs_to :manager, :class_name => "ExtManagementSystem"` at the base class so don't think you need this here.

Reply:
@agrare This was added for the corresponding chargeback PR - https://github.com/ManageIQ/manageiq/blob/8dd1508c931922c5592167188c2355ffd06a7e9e/app/models/chargeback/consumption_without_rollups.rb#L16

Reply:
Okay wonder if we should refactor that code, we handle `ext_management_system` vs `manager` in a few places.

If we need this to respond to `ext_management_system` we should `alias` this instead of creating a new `belongs_to` association.

Reply:
Thanks @agrare, i found a way to not use it.PR has been updated.